contracts/contracts/MACI.sol

```diff
@@ -38,31 +38,27 @@ contract MACI is IMACI, Params, Utilities, Ownable {
   mapping(uint256 => Poll) public polls;
 
   /// @notice The number of signups
-  uint256 public override numSignUps;
+  uint256 public numSignUps;
 
   /// @notice A mapping of block timestamps to the number of state leaves
   mapping(uint256 => uint256) public numStateLeaves;
 
   // ERC20 contract that hold topup credits
-  TopupCredit public topupCredit;
+  TopupCredit public immutable topupCredit;
```

Comment:
<img width="600" alt="image" src="https://github.com/privacy-scaling-explorations/maci/assets/9023842/fb7cdfa3-36f4-417d-9722-c520163b01c4">

Does this matter? or just leave it?

Reply:
I think we can can change at some other point

Reply:
Should we create an issue for this @kittybest @ctrlc03 @0xmad?

Reply:
Created on here: https://github.com/privacy-scaling-explorations/maci/issues/1012